### ISSUE TITLE
Non-HTML5 audio is not covered

### ISSUE THREAD

**Issue by ppseprus** [mavroudisv/Silverdog#15]
The method `document.getElementsByTagName('audio')` will only return HTML5 audio, but there could be audio embedded in Flash; there's Javascript audio object and God know what else...

**Comment by mavroudisv** [comment]
This is true, and it's a known limitation of the extension. Unfortunately, it seems impossible to handle all kinds of audio sources. However, flash is being phased out, and IIRC we do filter javascript audio objects.
